DeepLabCut/deeplabcut_parameters.py

```diff
@@ -0,0 +1,32 @@
+import sys
+sys.path.append("..")
+import utilities as utils
+import definitions as defs
+import DeepLabCut.deeplabcut_definitions as poseEst_defs
+
+class DeepLabCutParameters:
+
+    """
+    Parameters used in deepLabCut library
+    """
+
+    def __init__(self, danse_params: dict):
+        
+        self.paths: dict          = danse_params.get(defs.Parameters.Main.PATHS, {})
+        self.params: dict         = danse_params.get(defs.Parameters.Main.PARAMETERS, {})
+        #self.preview_params: dict = danse_params.get(defs.Parameters.Main.PREVIEW, {})
+
+    def get_h5path_names(self):
```

Comment:
Please edit this function taking into account that the path to the video is different acquisition group and behavior group